ar-setup.sh

```diff
@@ -110,9 +110,9 @@ add_gitlab_project_vars() {
 	https_prefix_len=${#https_prefix}
 	app_config_repo_without_https_prefix=${app_config_repo:https_prefix_len}
 
-	end_of_gitlab_hostname_index="$(expr index ${app_config_repo_without_https_prefix} /)"
-	gitlab_hostname_len=https_prefix_len+end_of_gitlab_hostname_index
-	gitlab_hostname=${app_config_repo:0:gitlab_hostname_len}
+	gitlab_hostname_without_http_prefix="$(echo ${app_config_repo_without_https_prefix} | cut -d'/' -f1)"
```

Comment:
I'm confused by the variable name. Are you going to update `app_config_repo_without_https_prefix`?
If not, where is `http_prefix` coming from?

Reply:
an example of `app_config_repo`: `https://gitlab.endpoints.app-delivery-platform-dev-1.cloud.goog/haiyan-group-3/sg1/sg2/myapp`

`app_config_repo_without_https_prefix` will be `gitlab.endpoints.app-delivery-platform-dev-1.cloud.goog/haiyan-group-3/sg1/sg2/myapp`

`gitlab_hostname_without_http_prefix` will be `gitlab.endpoints.app-delivery-platform-dev-1.cloud.goog`

`gitlab_hostname` will be `https://gitlab.endpoints.app-delivery-platform-dev-1.cloud.goog/`

Reply:
I see  where the confusion is coming from. I think you meant `gitlab_hostname_without_https_prefix` instead of `gitlab_hostname_without_http_prefix`, http vs. https.

Reply:
```suggestion
	gitlab_hostname_without_https_prefix="$(echo ${app_config_repo_without_https_prefix} | cut -d'/' -f1)"
```

Reply:
Nice catch. Fixed it! Thanks!